Tutorial.ipynb

```diff
@@ -1,1105 +1,532 @@
+        "id": "8781f9db"
+      },
+      "outputs": [],
+      "source": [
+        "!git clone -q https://github.com/PathwayCommons/pathway-abstract-classifier.git"
+      ],
+      "id": "8781f9db"
     },
-    "papermill": {
-     "duration": 0.040272,
-     "end_time": "2022-04-19T18:37:32.704797",
-     "exception": false,
-     "start_time": "2022-04-19T18:37:32.664525",
-     "status": "completed"
+    {
+      "cell_type": "code",
+      "execution_count": 12,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "4sTpfqU138VU",
+        "outputId": "f2473867-0452-4d77-910c-53f3eb9cdf0d"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "/content/pathway-abstract-classifier/pathway-abstract-classifier\n",
+            "archive  dev-requirements.txt  requirements.txt  Tutorial.ipynb\n",
+            "Data\t README.md\t       tests\n"
+          ]
+        }
+      ],
+      "source": [
+        "%cd pathway-abstract-classifier\n",
+        "!ls\n",
+        "!pip install -q -r requirements.txt\n",
+        "# Currently necessary to get this to run in Colab environment\n",
+        "!pip install -q --upgrade google-cloud-storage"
+      ],
+      "id": "4sTpfqU138VU"
     },
-    "tags": []
-   },
-   "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[[797  45]\n",
-      " [ 47 153]]\n",
-      "              precision    recall  f1-score   support\n",
-      "\n",
-      "       False       0.94      0.95      0.95       842\n",
-      "        True       0.77      0.77      0.77       200\n",
-      "\n",
-      "    accuracy                           0.91      1042\n",
-      "   macro avg       0.86      0.86      0.86      1042\n",
-      "weighted avg       0.91      0.91      0.91      1042\n",
-      "\n",
-      "0.7142926808037249\n"
-     ]
-    }
-   ],
-   "source": [
-    "# Check performance on validation set\n",
-    "from sklearn.metrics import accuracy_score\n",
-    "from sklearn.metrics import classification_report\n",
-    "from sklearn.metrics import matthews_corrcoef\n",
-    "from sklearn.metrics import confusion_matrix\n",
-    "\n",
-    "print(confusion_matrix((y_val),predictions))\n",
-    "print(classification_report((y_val), predictions))\n",
-    "print(matthews_corrcoef((y_val), predictions))"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "6a9840cb",
-   "metadata": {
-    "papermill": {
-     "duration": 0.015137,
-     "end_time": "2022-04-19T18:37:32.735337",
-     "exception": false,
-     "start_time": "2022-04-19T18:37:32.720200",
-     "status": "completed"
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c0ad5e77"
+      },
+      "source": [
+        "<h3> Below we import necessary libraries, turn on AMP (optional) and show two ways to load our model </h3>"
```

Comment:
This is an aesthetic thing but all your text blocks appear to be headers? I would probably just make them plain text and use headers more sparingly to group content/sections.